dafni/README.md

```diff
@@ -10,8 +10,8 @@ to upload the framework onto [DAFNI](https://www.dafni.ac.uk).
 - `data` contains two sub-folders (the structure is important for DAFNI).
   - `inputs` is a folder that contains the input files that are (separately) uploaded to DAFNI.
     - `causal_tests.json` is a JSON file that contains the causal tests.
-    - `variables.json` is a JSON file that contains the variables and constraints to be used.
-    - `dag.dot` is a dot file that contains the directed acyclc graph (dag) file.
+    - `dag.dot` is a dot file that contains the directed acyclc graph (dag) file, 
```

Comment:
Would it make sense to document the metadata attributes here?

Reply:
@jmafoster1 I was thinking of documenting this on the DAFNI platform itself, rather than in this README. Both?

Reply:
I think both is good. I've been using the `main_dafni.py` as a main entrypoint for other things to save me having to write a separate run script each time, so it makes sense to document it for users who might not have access to the DAFNI platform.

Reply:
In that case, would it make sense to have the dafni stuff be a separate repo? We may have had this discussion before, but if we only want the dafni stuff used on that platform, it probably doesn't make sense to distribute it with the CTF. Alternatively, a longer term goal could be to broaden the applicability of the entrypoint, but that doesn't invalidate this PR, so we can do that separately later.